web/src/components/profileEdit/QuotaCardEdit.tsx

```diff
@@ -51,9 +63,16 @@ const QuotaCardEdit: React.FC<IQuotaCardEditProps> = (props) => {
   const [goBackToProfileEditable, setGoBackToProfileEditable] = useState<boolean>(false);
   const [selectedSize, setSelectedSize] = useState<any>('');
   const [specs, setSpecs] = useState<any>([]);
```

Comment:
I also have noticed a bug that only seems to appear in this branch locally, if you manually goto a page that should be hidden (pending edit or not provisioned yet) I receive the following error:
```
TypeError: specs is undefined
QuotaCardEdit
src/components/profileEdit/QuotaCardEdit.tsx:122

  119 |   );
  120 | }
  121 | 
> 122 | if (specs.length === 0) {
      | ^  123 |   return null;
  124 | }
  125 | return (
  ```
  
  It would be good to see if you could replicate this or if it is just a local issue for me.
  
  To reproduce, create a new project and goto the project overview page. Manually update the URL from `.../:projectId/overview` to `.../:projectId/quota` and see if you receive a similar alarm.

Reply:
I tried on my localhost and docker and l can't reproduce this issue.

Reply:
Interesting, we can take a look when this is on DEV.